docs/user/README.md

```diff
@@ -0,0 +1,245 @@
+# ModelarDB Installation and Usage
+This document describes how to set up and use ModelarDB. Installation instructions are provided for 
+Linux, macOS, FreeBSD, and Windows. To support running ModelarDB in a containerized environment, instructions for 
+setting up a Docker environment are also provided. Once installed, using ModelarDB is consistent across all platforms.
+
+## Installation
+### Linux
+The following commands are for Ubuntu Server. However, equivalent commands should work for other Linux distributions.
+
+1. Install [build-essential](https://packages.ubuntu.com/jammy/build-essential): `sudo apt install build-essential`
+2. Install [CMake](https://cmake.org/): `sudo apt install cmake`
+
+### macOS
+1. Install the Xcode Command Line Developer Tools: `xcode-select --install`
+2. Install [CMake](https://cmake.org/) and follow the _"How to Install For Command Line Use"_ menu item.
+
+### FreeBSD
+1. Install [CMake](https://cmake.org/) as the *root* user: `pkg install cmake`
+2. Install [cURL](https://curl.se/) as the *root* user: `pkg install curl`
+
+### Windows
+1. Install a supported version of [Visual Studio](https://visualstudio.microsoft.com/vs/older-downloads/) with Visual C++:
+   - Visual Studio 2019 ([Supported](https://github.com/microsoft/snmalloc/blob/main/docs/BUILDING.md#building-on-windows))
```

Comment:
I think the `Supported` link helps since it links to the `Building on Windows` documentation where they mention that 2019 and 2022 are supported, so it makes it clear why the version is supported. 

Added link to MS's page.